<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgNewBill</class>
 <widget class="QWidget" name="CDlgNewBill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>585</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>新增订单</string>
  </property>
  <widget class="QPushButton" name="mopBtnNew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mopBtnSave">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mopBtnEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mopBtnVerify">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>审核</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mopBtnPrint">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mopBtnCancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabData">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabBillID">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>订单编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabClerk">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>业务员：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabDesigner">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>设计员：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabDeliveryWay">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>交货方式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabDeliveryData">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>交货日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabCustomerName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>客户名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabTel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>联系电话：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mopLineEditTel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabAddress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>联系地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mopLineEditAddress">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mopBtnSearch">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mopComBoxClerk">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>86</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="mopComBoxDesigner">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>117</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mopBtnAddItem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加明细</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mopBtnCopyItem">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>复制明细</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mopBtnDeleteItem">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除明细</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="mopTabItems">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>1131</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="mopDateOrderBuild">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>54</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="mopTimeDeliveryData">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>119</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="mopComBoxDeliveryWay">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>87</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="mopComCustomers">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mopLinEditOrderId">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>56</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabEarnestMoney">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>定金：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabTotalMoney">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>总价：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabDeliveryMoney">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>运费：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabOrderStatus">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="mopTxtRemarks">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>1121</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mopLabRemarks">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mopDeliveryPrice">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mopTotalPrice">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mopLinePrePay">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="mopComBoxOrderStatus">
   <property name="geometry">
    <rect>
     <x>514</x>
     <y>447</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
